scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -288,8 +293,14 @@ fi
 # shellcheck disable=SC1090
 source "./$projectFile"
 
-projectURL=$(get_local_ip_url "$(get_lan_ip)")
 
+
+
+if [[ -z "$COMMON_NAME" ]]; then
+	projectURL=$(get_local_ip_url "$(get_lan_ip)")
+else 
+	projectURL="https://$COMMON_NAME:$NGINX_HTTPS_PORT"
+fi
```

Comment:
I don't see any reference to `COMMON_NAME` anywhere and generating the URL with port, IP and protocol by hand is brittle and error prone.  I think we should remove this section.  

If you feel different, lemme know!

Reply:
COMMON_NAME is use in case of AUTOI_GENERATE tls certificate, DOMAIN might be a better choice here

Reply:
Ah - I think i see a bit better.

Is your thinking that you don't want to use `127-0-0-1.local-ip.medicmobile.org` which `get_lan_ip()` will fall back to and want to explicitly specify it when you make the CLI call?  Given we [only work with `*.local-ip.medicmobile.org` certificates](https://github.com/medic/cht-core/blob/4a51fd6cf92002361d79008f7ff5ac2385a0d58f/scripts/docker-helper-4.x/cht-docker-compose.sh#L322), maybe we could go with something like this:

```suggestion
if [[ -z "$OVERRIDE_IP" ]]; then
	projectURL=$(get_local_ip_url "$(get_lan_ip)")
else 
	projectURL=$(get_local_ip_url ${OVERRIDE_IP})
fi
```